.travis.yml

```diff
@@ -20,7 +20,7 @@ before_install:
       tar -zxf "stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}_linux_amd64.tar.gz" -C "stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}/"
     fi
   - |
-    stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}/stripe-mock -https > /dev/null &
+    stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}/stripe-mock -https-port 12112 > /dev/null &
```

Comment:
You're missing a space character before the `-port` option.

Reply:
Doh, okay sorry this new set of options can be a little bit confusing, but in this case it's actually allowed to be either `-https -port 12112` _or_ `-https-port 12112`. The latter is required if you want simultaneous HTTP and HTTPS, but it can also be used by itself.

Reply:
Ah right, of course 🤦‍♂️